io/LasWriter.cpp

```diff
@@ -485,6 +488,8 @@ void LasWriter::addSpatialRefVlrs()
     deleteVlr(las::TransformUserId, las::GeotiffAsciiRecordId);
     deleteVlr(las::TransformUserId, las::WktRecordId);
     deleteVlr(las::LiblasUserId, las::WktRecordId);
+    // deleteVlr(las::PdalUserId, las::PdalWkt2RecordId);
+    // deleteVlr(las::PdalUserId, las::PdalProjJsonRecordId);
```

Comment:
I commented them out for test `read_consume_order`. Is there a better way to do it?